internal/cmd/cmd.go

```diff
@@ -132,7 +132,7 @@ func listTasks(args []string, t *task.Task) error {
 	}
 
 	if len(args) > 2 {
-		return errors.New("listing tasks only requires 2 arguments")
+		return errors.New("list tasks only requires 2 arguments")
```

Comment:
list, which is a command, actually takes only up to one argument, which is task status. You should always look at errors from user perspective, not from code.